package.json

```diff
@@ -15,32 +15,40 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/lab": "^4.0.0-alpha.56",
     "@react-firebase/auth": "^0.2.10",
-    "@react-spectrum/combobox": "^3.0.0-alpha.2",
+    "@react-spectrum/color": "^3.0.0-beta.1",
+    "@react-spectrum/combobox": "^3.0.0-beta.0",
+    "@react-spectrum/slider": "^3.0.0",
+    "@react-spectrum/table": "^3.0.0-alpha.9",
     "@react-spectrum/tabs": "^3.0.0-alpha.3",
     "@spectrum-css/typography": "^3.0.0-beta.0",
     "@spectrum-icons/illustrations": "^3.2.1",
     "@spectrum-icons/workflow": "^3.2.0",
+    "@spectrum-web-components/accordion": "^0.1.4",
```

Comment:
react spectrum accordion also in alpha state...use material accordion @Khushbu

Reply:
done